launch/tier4_map_launch/launch/map.launch.py

```diff
@@ -57,7 +62,10 @@ def launch_setup(context, *args, **kwargs):
         package="map_loader",
         plugin="Lanelet2MapLoaderNode",
         name="lanelet2_map_loader",
-        remappings=[("output/lanelet2_map", "vector_map")],
+        remappings=[
+            ("output/lanelet2_map", "vector_map"),
+            ("input/map_projector_info", "map_projector_type"),
```

Comment:
@kminoda 
Why did you use "input/map_projector_info" instead of "input/map_projector_type"?
I believe using the same name "map_projector_info" as the message type would be more intuitive.

Reply:
Thank you! I understood that it's rather a question to the current implementation of Autoware.
@mitsudome-r @tkhmy Do you know anything about this?

Reply:
I don't think there are any reasons. I also think projector info is better as well.

Reply:
I think initially I just publish mgrs, so I use type.
Then after that the topic also publish information, so change to info , so I think /map/map_projector_info will be a better naming

Reply:
Thank you for the info.
How about this?
- This PR will be merged as it is
- The topic name will be changed in another PR after this

Reply:
OK!